src/executor/LanceurExecutor.java

```diff
@@ -32,7 +32,7 @@ public Integer call() throws Exception {
 		try {
 			Integer retour = f.get();
 			System.out.printf("j'ai eu le retour %s%n", retour);
-
+                        System.out.println("Coucou sonar !");
 		} catch (InterruptedException e) {
 			e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)